janus_core/cli/md.py

```diff
@@ -133,6 +133,20 @@ def md(
     ] = 10,
     post_process_kwargs: PostProcessKwargs = None,
     correlation_kwargs: CorrelationKwargs = None,
+    plumed_input: Annotated[
+        str | None,
+        Option(
+            help="PLUMED input script content or path to PLUMED input file.",
+            rich_help_panel="Calculation",
+        ),
+    ] = None,
```

Comment:
Will this ever realistically be expressible as a plain string? Maybe should just be a `Path` here?